backend/api/Database/Models/Pose.cs

```diff
@@ -62,7 +63,22 @@ public class Pose
 
         [MaxLength(64)]
         public string Frame { get; set; }
-
+        private static float AxisAngleToQuaternionElement(float coordinate, float angle)
```

Comment:
What is "coordinate" here? Should it be named "direction" or something else?

Reply:
rotationAxis?

Reply:
axis or rotationAxis